src/App.js

```diff
@@ -1,26 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] It would be better if your app component is class-based. Kindly make this a minor issue. 😄 👍 

```suggestion
```javascript
class App extends Component {
  
}
```
```